Example/OriginateUI/InstallationViewController.m

```diff
@@ -0,0 +1,178 @@
+//
+//  InstallationViewController.m
+//  OriginateUI
+//
+//  Created by Seth Rainha on 6/22/16.
+//  Copyright © 2016 originate.com. All rights reserved.
+//
+
+#import "InstallationViewController.h"
+#import "CodeViewController.h"
+@import OriginateUI;
+
+@interface InstallationViewController ()
+
+@property (nonatomic, strong) UILabel *instLabel;
+@property (nonatomic, strong) UILabel *reqLabel;
+@property (nonatomic, strong) UILabel *useLabel;
+@property (nonatomic, strong) UILabel *instLabel2;
+@property (nonatomic, strong) UILabel *reqLabel2;
+@property (nonatomic, strong) UILabel *useLabel2;
+@property (nonatomic, strong) UILabel *endLabel;
+@property (nonatomic, strong) UITextView *textViewFirst;
+@property (nonatomic, strong) UITextView *textViewSecond;
+
+@end
+
+@implementation InstallationViewController
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    self.view.backgroundColor = [UIColor whiteColor];
+    self.title = @"Installation";
+    [self.view addSubview:self.instLabel];
+    [self.view addSubview:self.reqLabel];
+    [self.view addSubview:self.useLabel];
+    [self.view addSubview:self.instLabel2];
+    [self.view addSubview:self.reqLabel2];
+    [self.view addSubview:self.useLabel2];
+    [self.view addSubview:self.endLabel];
+    [self.view addSubview:self.textViewFirst];
+    [self.view addSubview:self.textViewSecond];
+}
+
+- (UILabel *)instLabel
+{
+    if (!_instLabel){
+        CGFloat screenHeight = CGRectGetHeight(self.view.frame);
+        CGFloat screenWidth = CGRectGetWidth(self.view.frame);
+        self.instLabel = [[UILabel alloc] initWithFrame:CGRectMake(10.0, screenHeight*0.2, screenWidth*0.8, 50.0)];
+        self.instLabel.text = @"Install with CocoaPods";
+        self.instLabel.font = [UIFont fontWithName:@"CircularPro-Medium" size:22];
+        self.instLabel.numberOfLines = 0;
```

Comment:
What's the line break mode here? If you set `0` to `numberOfLines`, how many lines would the shown?